analyses/admin/study.py

```diff
@@ -336,6 +338,173 @@ def make_state_link(state: Analysis.AnalysisStates) -> str:
+                ]
+            )
+            running_count = sum(
+                [
+                    batch.pipeline_status_counts.asa.running,
+                    batch.pipeline_status_counts.virify.running,
+                    batch.pipeline_status_counts.map.running,
+                ]
+            )
+            failed_count = sum(
+                [
+                    batch.pipeline_status_counts.asa.failed,
+                    batch.pipeline_status_counts.virify.failed,
+                    batch.pipeline_status_counts.map.failed,
+                ]
+            )
+            completed_count = sum(
+                [
+                    batch.pipeline_status_counts.asa.completed,
+                    batch.pipeline_status_counts.virify.completed,
+                    batch.pipeline_status_counts.map.completed,
+                ]
+            )
+
+            # Add to totals
+            total_pending += pending_count
+            total_running += running_count
+            total_failed += failed_count
+            total_completed += completed_count
+
+            row = [
+                format_html(
+                    "<a href='{}'>{}</a>",
+                    reverse_lazy(
+                        "admin:workflows_assemblyanalysisbatchanalysis_changelist",
+                        query={"batch__id__exact": str(batch.id)},
+                    ),
+                    str(batch.id)[:8],
+                ),
+                batch.total_analyses,
+                pending_count,
+                running_count,
+                failed_count,
+                completed_count,
+                batch.updated_at.strftime("%Y-%m-%d %H:%M"),
+            ]
+            rows.append(row)
+
+        batches_table = {
+            "headers": [
+                "Batch",
+                "Total analyses",
+                "Pending analyses",
+                "Running analyses",
+                "Failed analyses",
+                "Completed analyses",
+                "Updated",
+            ],
+            "rows": rows,
+        }
+
+        # Calculate total analyses across all batches
+        total_analyses = sum(batch.total_analyses for batch in batches)
+
+        # Create summary card items with links
+        summary_items = [
+            {
+                "label": "Total analyses",
+                "value": total_analyses,
+                "colour": "white",
+                "url": reverse_lazy(
+                    "admin:workflows_assemblyanalysisbatchanalysis_changelist",
+                    query={"study__accession__exact": study.accession},
+                ),
+            },
+            {
+                "label": "Pending",
+                "value": total_pending,
+                "colour": "blue",
+                "url": reverse_lazy(
+                    "admin:workflows_assemblyanalysisbatchanalysis_changelist",
+                    query={
+                        "batch__study__id__exact": study.id,
```

Comment:
Wouldn't this return only analysis for which all 3 statuses are pending ?

Reply:
Yeah, "an any of the them pending would be better". I'll try to adjust this